<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;EXPORT SETTINGS - (Asset Collection Name)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QListWidget" name="asset_col_list">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <item>
         <property name="text">
          <string>(select all)</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blocks</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Centroids</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flowpaths</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="export_options_widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="0">
          <widget class="QLabel" name="export_rasterfmt_lbl">
           <property name="text">
            <string>Default Raster Format:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QRadioButton" name="exportdir_project_radio">
           <property name="text">
            <string>Project Folder</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="exportdir_line"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="export_metadata_check">
           <property name="text">
            <string>Write metadata file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="export_filename_line"/>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="export_seriesfmt_combo">
           <item>
            <property name="text">
             <string>.csv</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>.txt</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="12" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QCheckBox" name="exportdir_open_check">
           <property name="text">
            <string>Open directory after asset export</string>
           </property>
           <property name="icon">
            <iconset resource="../graphics/ubeats.qrc">
             <normaloff>:/icons/folder-open-icon.png</normaloff>:/icons/folder-open-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="export_seriesfmt_lbl">
           <property name="text">
            <string>Default Series Format:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="export_rasterfmt_combo">
           <item>
            <property name="text">
             <string>GeoTIFF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ASCII</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="export_filename_lbl">
           <property name="text">
            <string>File naming convention:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="exportdir_lbl">
           <property name="text">
            <string>Output Directory:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QRadioButton" name="exportdir_custom_radio">
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="export_csvtable_check">
           <property name="text">
            <string>Export attribute tables as .csv</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="asset_col_combo">
        <item>
         <property name="text">
          <string>(select asset collection)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="asset_col_list_title">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select Assets to export...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="export_options_title">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Export Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="export_button">
        <property name="text">
         <string>Export</string>
        </property>
        <property name="icon">
         <iconset resource="../graphics/ubeats.qrc">
          <normaloff>:/icons/export-icon.png</normaloff>:/icons/export-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_button">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="../graphics/ubeats.qrc">
          <normaloff>:/icons/delete-icon.png</normaloff>:/icons/delete-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../graphics/ubeats.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
